packages/angular_devkit/benchmark/src/main.ts

```diff
@@ -69,13 +70,23 @@ export async function main({
     'watch-timeout': number;
     'watch-matcher'?: string;
     'watch-script'?: string;
-    '--': string[] | null;
+    '--': string[];
+    _: string[];
+    $0: string;
   }
 
   // Parse the command line.
-  const argv = minimist(args, {
+  const argv = yargsParser(args, {
     boolean: ['help', 'verbose', 'overwrite-output-file'],
     string: ['watch-matcher', 'watch-script'],
+    configuration: {
+      'unknown-options-as-args': false,
```

Comment:
It looks like `unknown-options-as-args` is false by default.
https://github.com/yargs/yargs-parser/tree/0fdbfa1fe960f76d520f75437e97c79727804908#unknown-options-as-args

Reply:
Same as previous.